metadata.rb

```diff
@@ -27,3 +27,4 @@
 source_url 'https://github.com/sensu/sensu-go-chef'
 
 depends 'packagecloud'
+depends 'chocolatey'
```

Comment:
According to the docs `chocolatey_package` is a native resource since chef client 12.7: https://docs.chef.io/resources/chocolatey_package/

Was there a specific feature that needed the cookbook include?

Reply:
This was to install the Chocolatey resource manager. We can remove that, and there should be no other issues aside form noting that users would need to install it outside of this cookbook, since we're not using it as a dependency.

Reply:
I could go either way so it's not a blocking issue for me, I usually prefer to keep dependencies minimal but I'll leave final call up to @majormoses

Reply:
Noting that `chocolatey` needs to be installed by an assumed wrapper is a poor implementation, if removed, the recipe would fail to converge.

Reply:
@derekgroh so if I understand correctly this basically comes down to whether or not we should assume chocolatey is already on the system? If that is the case I agree that adding the dependency makes sense, I would add a comment as to the rationale of this. Maybe someday chef will have the concept of dynamic dependencies.
```suggestion
# since it may not be installed on the system
depends 'chocolatey'
```

Reply:
btw I really wish chef had a way of saying `depends 'chocolatey' if platform.windows?` but I know that the deps are evaluated before it really reaches the node.

Reply:
Me too. This would be so useful in this case where dependencies are conditional per platform